src/translations/en.json

```diff
@@ -3184,11 +3184,11 @@
         },
         "system_health": {
           "caption": "System Health",
-          "description": "Status, Stats and Integration startup time",
+          "description": "Status, statistics and integration startup time",
```

Comment:
"statistics" is the proper word and less informal

Reply:
But confusing, as we have something else that is also called statistics

Reply:
How about metrics?

Reply:
We should use the same title that is on the page. Currently that is `Stats`.
<img width="396" alt="CleanShot 2022-05-04 at 12 40 54@2x" src="https://user-images.githubusercontent.com/20203284/166666601-38baf565-1374-482b-b352-df9d2646b17c.png">

Reply:
> We should use the same title that is on the page. Currently that is `Stats`. 

That's why I had changed those in this PR as well 😉. I agree with "metrics". Will adjust this PR.